core/src/main/java/org/apache/calcite/adapter/jdbc/JdbcRules.java

```diff
@@ -426,7 +426,7 @@ protected JdbcJoin(
     private JdbcCalcRule(JdbcConvention out,
         RelBuilderFactory relBuilderFactory) {
       super(Calc.class, (Predicate<RelNode>) r -> true, Convention.NONE,
-          out, relBuilderFactory, "JdbcCalcRule");
+          out, relBuilderFactory, "JdbcCalcRule:" + out);
```

Comment:
Can we have a uniform method to generate the name pattern for JDBC rules ? Because the pattern rules for all the rules are the same.